src/app/core/admin/user-details/user-details.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { User } from '../../user/user';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { EntityMapperService } from '../../entity/entity-mapper.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'app-user-details',
+  templateUrl: './user-details.component.html',
+  styleUrls: ['./user-details.component.scss'],
+})
+export class UserDetailsComponent implements OnInit {
+  creating: boolean = true;
+  private readonly defaultPasswordValue: string = 'password';
```

Comment:
I wouldn't set defaults (only placeholders as hints for the user). Let's better leave the form invalid until the user enters something properly.